dev_docs/getting_started/setting_up_a_development_env.mdx

```diff
@@ -46,6 +46,13 @@ Then, install the latest version of yarn using:
 npm install -g yarn
 ```
 
+If you already have Yarn installed, make sure its version is 1.22.19 or newer. Versions 2+ of Yarn are incompatible with this project.
+
+In order to set your previously installed Yarn to a compatible version (for example 1.22.19) run
+```sh
```

Comment:
```suggestion
[source,text]
--
```

Reply:
Hey, @KOTungseth . I don't see any ocurrence of [source,text] in *.mdx files on the Kibana repo. Only in *.asciidoc. Sorry for the rookie question, but what would it do in *.mdx files?

Reply:
My apologies! You're right. Please ignore my comment.